app/models/hydra/datastream/rights_metadata.rb

```diff
@@ -0,0 +1,270 @@
+require 'active_support/core_ext/string'
```

Comment:
Why is this file needed? Is it because it was defined in hydra-head, and hydra-access-controls depends on it?

Reply:
@jcoyne 	Worth a shot. Have a preference for doing that in this branch or in follow-up work?

Reply:
I'd like to do that elsewhere.  I prefer to keep PRs small as possible.

Reply:
You bet.